File: 0026-remove-duplicates-from-sorted-array/solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ans=0
        length = len(nums)
        for i in range(1, length):
            if nums[i]!=nums[ans]:
                ans+=1
                nums[ans] = nums[i]

        return ans+1

File: 0026-remove-duplicates-from-sorted-array/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ind=0;
        // int dup=1;
        int n=nums.size();
        for(int i=1;i<n;i++) {
            if(nums[ind]!=nums[i]) {
                ind++;
                nums[ind]=nums[i];
            }
        }
        return ind+1;
    }
};
